include/envoy/stream_info/stream_info.h

```diff
@@ -65,6 +65,14 @@ enum ResponseFlag {
   LastFlag = StreamIdleTimeout
 };
 
+/**
+ * Constants for the response code details field of StreamInfo.
+ *
+ * These provide details about the stream state such as whether the
```

Comment:
One other minor nit - do you think it's worth clarifying what's done now vs what this will do?
These will provide such details, but right now they only do rc_set_by_upstream

I'd be fine with a tracking issue assigned my way for actually setting values in sendLocalReply and referencing the issue here?

Reply:
Filed #6542 to track this. I'll add a reference to this in the code.